package.json

```diff
@@ -41,7 +41,7 @@
     "ensure-deps": "./scripts/ensure-dependencies.sh",
     "get-next-version": "node scripts/get-next-version.js",
     "postinstall": "node ./scripts/run-postInstall.js",
-    "lint": "lerna run lint --no-bail --concurrency 2 && eslint --ext .js,.ts,.json, scripts",
```

Comment:
the linting of the `/scripts` folder is now moved inside of `/scripts/package.json`

Reply:
this dropped linting of the root-level js/json files which we want to keep